.github/workflows/cancel-duplicate-runs.yaml

```
@@ -8,7 +8,8 @@ jobs:
   cancel:
     name: Cancel previous runs
     runs-on: ubuntu-latest
+    if: github.repository == 'xarray-contrib/xskillscore'
```

mind keeping black like quotations
```suggestion
    if: github.repository == "xarray-contrib/xskillscore"
```